src/components/workspace-utils.js

```diff
@@ -39,10 +39,16 @@ export const useWorkspaces = (fieldsArg, stringAttributeMaxLength) => {
     const ws = await Ajax(signal).Workspaces.list(fields, stringAttributeMaxLength);
     workspacesStore.set(ws);
   });
+
+  const refreshSilently = _.flow(withErrorReporting('Error loading workspace list'))(async () => {
+    const ws = await Ajax(signal).Workspaces.list(fields, stringAttributeMaxLength);
```

Comment:
does the refresh need to get all of these fields or can we limit it to the workspace's state?

Reply:
... and the error message I suppose. Just wondering if we can limit this call to only the workspace fields that are expected to change rather than using the same set of fields we load with in the first place. Not sure it's worth the extra effort when it comes to updating `workspacesStore`